packages/storybook/src/stories/Filters/organisms/FilterBar.stories.tsx

```diff
@@ -114,14 +114,14 @@ export const _FilterBar = () => {
   const [data, setData] = useState<ITableSampleData[]>(dataInitialState);
   const [rows, setRows] = useState<ITypeTableData>(rowMaker(dataInitialState));
   const [filters, setFilters] = useState<IFilterResult[]>([]);
-
   /**
  * Story interaction section
  */
   const singleFilter = boolean('Single Filter', false);
   const hasShowMore = boolean('Has Show More', true);
   // valid formats - https://date-fns.org/v2.25.0/docs/format
   const resultsDateFormat = text('Results date format', 'yyyy-MM-dd HH:mm');
+  const datePickerHasApply = select('Datepicker has Apply', { true: true, false: false }, true);
```

Comment:
Could be a boolean similar to `singleFilter` and `hasShowMore` do above, for consistency.

Reply:
Updated